src/main/java/nextstep/mission/domain/Point.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.mission.domain;
+
+import java.util.Objects;
+
+public class Point {
+    private boolean point;
+
+    public Point(boolean point) {
+        this.point = point;
+    }
+
+    public boolean isRight(Point right) {
+        if (right == null)
+            return false;
+
+        return right.toBoolean();
+    }
```

Comment:
상태값인 point값을 사용하지 않는 메소드네요.
그렇다면 인자를 받지 않고 아래처럼 고쳐도 무방하지 않을까요?
```suggestion
    public boolean isRight() {
       return point;
    }
```
(point의 값이 오른쪽의 가로라인?을 의미한다면요 :) )

Reply:
체이닝 메소드로 작업을 해보려다가 그 부분까지는 완성을 못했습니다...